crates/test-collector/src/lib.rs

```diff
@@ -215,6 +233,23 @@ pub fn try_extract_test_config(
     } else {
         (false, None)
     };
+    let fork_config = if let Some(attr) = fork_attr {
+        if attr.args.is_empty() {
+            None
+        } else {
+            extract_fork_config(db, attr).on_none(|| {
+                diagnostics.push(PluginDiagnostic {
+                    stable_ptr: attr.args_stable_ptr.untyped(),
+                    message: "Expected fork config must be of the form `url: <double quote \
+                                  string>, block_id: <snforge_std::BlockId>`."
+                        .into(),
```

Comment:
Should we have a test that checks how this error is displayed?

Reply:
done